model/src/main/kotlin/AdvisorRecord.kt

```diff
@@ -61,6 +61,32 @@ data class AdvisorRecord(
             results.any { it.summary.issues.isNotEmpty() }
         }
     }
+
+    /**
+     * Return a list with all [Vulnerability] objects that have been found for the given [package][pkgId]. Results
+     * from different advisors are merged if necessary.
+     */
+    fun getVulnerabilities(pkgId: Identifier): List<Vulnerability> {
+        return advisorResults[pkgId]?.flatMap { it.vulnerabilities }?.mergeVulnerabilities() ?: emptyList()
```

Comment:
Nit: Could be a function expression. Also, does `orEmpty()` work here?

Reply:
starting with `return advisorResults[pkgId].orEmpty().flatMap...` should make it a bit more compact.